R/autoplot.pop_data.R

```diff
@@ -73,7 +89,7 @@ age_scatter <- function(
     ggplot2::theme_linedraw() +
     # ggplot2::scale_y_log10() +
 
-    # avoid log 0 (https://forum.posit.co/t/using-log-transformation-but-need-to-preserve-0/129197/4)
+    # avoid log 0 (https://bit.ly/4eqDkT4)
```

Comment:
Shorten the URL to reduce line to 80 characters so as to match linting requirement.